File: test/fixtures/release/typescript/src/pricing.js
export function discountedPrice(price, percentage) {
  if (!Number.isFinite(price) || price < 0) {
    throw new TypeError("price must be a non-negative finite number");
  }
  if (!Number.isFinite(percentage) || percentage < 0 || percentage > 100) {
    throw new TypeError("percentage must be between zero and one hundred");
  }
  return price * (1 - percentage / 100);
}

File: test/fixtures/release/python/src/pricing.py
def discounted_price(price: float, percentage: float) -> float:
    if price < 0:
        raise ValueError("price must be non-negative")
    if percentage < 0 or percentage > 100:
        raise ValueError("percentage must be between zero and one hundred")
    return price * (1 - percentage / 100)
